gdk4-macos/Cargo.toml

```
@@ -15,13 +15,15 @@ version.workspace = true
 
 [features]
 v4_8 = ["gdk4-macos-sys/v4_8"]
+cocoa = ["cocoa"]
```

```suggestion
cocoa = ["dep:cocoa"]
```
or so, the new feature has to be also added to https://github.com/gtk-rs/gtk4-rs/tree/main/gdk4-macos#features

---

ah! nice thanks

---

done